gh-80010: Expand fromisoformat to include most of ISO-8601 (#92177)
python/cpython: Doc/whatsnew/3.11.rst

  existing stream-based connections to TLS. (Contributed by Ian Good in
  :issue:`34975`.)

datetime
--------

* :meth:`datetime.date.fromisoformat`, :meth:`datetime.time.fromisoformat` and

>   :meth:`datetime.datetime.fromisoformat` can now be used to parse most ISO 8601
>   formats (barring only those that support fractional hours and minutes).

Reviewer: A minor nit, but since the changelog is read by a lot of non native speakers, it seems beneficial to use simpler English here.
```suggestion
  :meth:`datetime.datetime.fromisoformat` can now be used to parse all ISO 8601
  formats except those that use fractional hours or minutes.
```

>   (Contributed by Paul Ganssle in :gh:`80010`.)

Reviewer: 80010 is the issue, 92177 is the PR - shouldn't this be the PR, instead?

Author: The other entries seem to reference the issue.
